app/models/list_item.rb

```diff
@@ -20,5 +20,5 @@ class ListItem < ApplicationRecord
 
   validates :book_id, presence: true
   validates :user_id, presence: true
-  validates :rating, presence: true, numericality: { in: 1..5 }
+  validates :rating, presence: true, numericality: { in: 0..5 }
```

Comment:
`nit`

can make use of `greater_than` and `less_than`, instead of making rails iterate over a range of values. A bit more performant. But again , this is a `nit`. Performance is negligible